pydy/viz/scene.py

```diff
@@ -433,13 +448,13 @@ def remove_static_html(self, force=False):
 
         if not force:
             ans = raw_input("Are you sure you would like to delete the " +
-                                "'static' directory? [y|n]\n")
+                            "'static' directory? [y|n]\n")
```

Comment:
The directory is called `static` in the source tree and that's fine. But maybe it should be called something like `pydy_app` or something when it is copied to cwd. I feel like a more unique name is in order.